napari_cellseg3d/code_plugins/plugin_model_training.py

```diff
@@ -1246,8 +1246,9 @@ def on_finish(self):
         self.log.print_and_log("Done")
         self.log.print_and_log("*" * 10)
         try:
-            self._make_csv()
-        except ValueError as e:
+            pass  # FIXME
+            # self._make_csv()
```

Comment:
Should this be fixed before merging? Is it saved below as zip file?

Reply:
Right, thank you, no it's not saved but I was thinking that maybe WandB integration could kind of replace this in a better way. It's just the loss values added as csv.

I will try to fix the issue regardless, it was caused by a wrong indexing I think